### ISSUE TITLE
Can't use with -Executable gitversion -Package gitversion.portable

### ISSUE THREAD

**Issue by Jaykul** [Jaykul/ChoVersion#2]
`Get-ChoVersion` is using the `$Executable` parameter, but `Set-ChoVersion` is not, so we can't install gitversion, because the package name is not the same as the executable name.

Repro:

```PowerShell
Set-ChoVersion GitVersion.Portable -Executable GitVersion
```
```Text
Error: Get-Command : The term 'gitversion.portable' is not recognized as the name of a cmdlet, function, script file, or operable program.
```

After that error, `Get-ChoVersion` will work, but `Set-ChoVersion` still fails.

![image](https://user-images.githubusercontent.com/192942/142264059-0301cdd6-87b1-4285-85c7-f9a0b23e8fc2.png)

**Comment by Jaykul** [comment]
The fixed version complains (when using a ChoVersion.psd1) about apps that don't have "executable" set.

Specifically, when using this ChoVersion.psd1
```powershell
@{
    ChocolateyPackages = @(
        @{
            Package = "bicep"
            Version = "0.4.1008"
        }
        @{
            Executable = "gitversion"
            Package = "gitversion.portable"
            Version = "5.8.1"
        }
    )
}
```

I got this error:

VERBOSE: Setting choco package 'bicep' version 0.4.1008
ERROR: Get-Command : Cannot validate argument on parameter 'Name'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.

NOTE: as a work around, I just set `Executable = "bicep"` on the first package and that worked.

Have a look at `Get-ChoVersion`, because that one is handling this properly.
